### ISSUE TITLE
Close Cockpit

### ISSUE THREAD

**Issue by ibeglab** [H-J-P/Ikarus#3]
If I close the cockpit window by using "Close" button, in the Ikarus main window the Show/Close Cockpit button is not synchronized. It shows "Close Cockpit", but the cockpit window is already closed, and if I click it the cockpit window opens again.

**Comment by H-J-P** [comment]
thats true. There is no interaction between the 2 windows. If you do it over the Main GUI, than it works. But it works.


CU Heinz

PS: ... die 1000 Kleinigkeiten ... und halte es einfach (keep it simple) ...

heinz.joerg.puhlmann@googlemail.com
